### PR TITLE
[FIX] account: Avoid ir_module.account_templates compute error

### PR DESCRIPTION
**Pull request by antoine162**
Issue / How to reproduce:

At the moment, if a localization's python module is not present, the computation of the `ir.module.module.account_templates` field fails with a ModuleNotFoundError

How to reproduce:
- Create a fresh DB with a localization e.g. l10n_be
- Remove the localization directory
- Start the DB again and navigate e.g. to Inventory > Configuration > Settings. You will see a traceback, whose root cause is that the computation of the `account_templates` field failed.

We noticed this in a runbot upgrade between 16.1 and 16.2, for a new localization that was introduced in 16.0.

Solution:

Make the computation return False if the Python module cannot be found. Also, filter out the modules where account_templates is False when constructing the list of available chart templates.

### CODE REVIEW

File: addons/account/models/ir_module.py [hunk 1]
```diff
@@ -35,7 +35,11 @@ def _compute_account_templates(self):
         for module in self:
             templates = {}
             if module.category_id == chart_category or module.name == 'account':
-                python_module = import_module(f"odoo.addons.{module.name}.models")
+                try:
+                    python_module = import_module(f"odoo.addons.{module.name}.models")
+                except ModuleNotFoundError:
+                    module.account_templates = False
+                    continue
```
Comment (william-andre) [inline addons/account/models/ir_module.py:42]: I find something like this more readable
```python
                try:
                    python_module = import_module(f"odoo.addons.{module.name}.models")
                except ModuleNotFoundError:
                    templates = {}
                else:
                    templates = {
                        fct._l10n_template[0]: {
                            'name': fct(ChartTemplate).get('name'),
                            'parent': fct(ChartTemplate).get('parent'),
                            'country': fct(ChartTemplate).get('country', ''),
                            'visible': fct(ChartTemplate).get('visible', True),
                            'module': module.name,
                        }
                        for _name, mdl in getmembers(python_module, template_module)
                        for _name, cls in getmembers(mdl, template_class)
                        for _name, fct in getmembers(cls, template_function)
                    }
```

Then you don't need to change anything in `addons/account/models/chart_template.py`

Comment (antoine162) [reply addons/account/models/ir_module.py]: oh yes, it is much neater indeed

**Comment by antoine162** [comment]
in my opinion it's good, I've tested it, it solves the problem of the computed field not being able to be computed.

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**